docs/secure-connections/sauce-connect/setup-configuration/specialized-environments.md

```diff
@@ -64,6 +64,18 @@ In this example, we'll [set our credentials (username/access key) as environment
 ./sc -u $SAUCE_USERNAME -k $SAUCE_ACCESS_KEY -r us-west --tunnel-name rdc-on-sauce-tunnel-us
 ```
 
+:::warning Note for Android Devices
+If you set up the tunnel for Android Devices, you have to start Sauce Connect Proxy with the extra flag `-B all`:
+
+```bash
+./sc -u $SAUCE_USERNAME -k $SAUCE_ACCESS_KEY -r $SAUCE_DATA_CENTER --tunnel-name $TUNNEL_NAME -B all
+```
+
+You need the extra flag because, by default, traffic on Sauce Connect is re-encrypted using Sauce Labs' self-signed certificates. While this can be trusted on Sauce Labs Virtual Cloud and Sauce Labs iOS devices, due to security specifications by Google, the same can't be done on Android.
```

Comment:
```suggestion
You need to include the -B all flag because, by default, traffic on Sauce Connect is re-encrypted using Sauce Labs' self-signed certificates. While this can be trusted on Sauce Labs Virtual Cloud and Sauce Labs iOS devices, due to security specifications by Google, the same can't be done on Android.
```